src/bp/sdk/botpress.d.ts

```diff
@@ -917,6 +917,8 @@ declare module 'botpress/sdk' {
       handler: MiddlewareHandler
       /** Indicates if this middleware should act on incoming or outgoing events */
       direction: EventDirection
+      /** Allows to override the middleware chain timeout limit */
+      timeout?: string
```

Comment:
Technically, it overrides the timeout of that specific middleware, not the whole chain. Maybe we could add a `@default 2s` value to give an example of what kind of value should go there?

Reply:
I agree an example would be useful. I wasn't sure what value to put there